tox.ini

```diff
@@ -68,12 +67,9 @@ passenv =
     RUN_REAL_PEBBLE_TESTS
     PEBBLE
 deps =
-    pytest
-    ipdb
     ipython!=8.1.0 # this version is broken and was causing failures
-    logassert
     coverage[toml]
-    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/requirements-dev.txt
```

Comment:
Any reason why you've moved most of these but not ipython and coverage? IMO we get rid of ipython in any case. I can't understand why ipython is needed for running these tests.

Reply:
yeah, I really wasn't sure about `ipython` and where it's being used. For coverage, I viewed it similar to `ipdb` in that it's one of the many debugging tools that a user(a dev) can choose from. 

So within the dev environment a user may choose to use a different coverage plugin, different from that contained in the `tox` script.

However, I do also believe this is a rather fringe case and this isn't a strongly held opinion of mine. So I'll move it all into `requirements-dev.txt` for the sake of uniformity/consistency.